app/commands/source/add_value_command.rb

```diff
@@ -0,0 +1,99 @@
+# frozen_string_literal: true
+
+module Source
+  class AddValueCommand < ApplicationCommand
+    usage do
+      no_command
+    end
+
+    keyword :name do
+      desc "Name for the new value"
+      required
+    end
+
+    keyword :attribute_friendly_id do
+      desc "Friendly ID of the primary attribute to add the value to"
+      required
+    end
+
+    def execute
+      frame("Adding new value") do
+        ActiveRecord::Base.transaction do
+          setup!
+          create_value!
+          create_attributes_value!
+        rescue ActiveRecord::Rollback => e
+          logger.fatal(e.message)
+          exit(1)
+        end
+
+        update_data_files!
+      end
+    end
+
+    private
+
+    def setup!
+      @primary_attribute = Attribute.find_by(friendly_id: params[:attribute_friendly_id])
+
+      raise ActiveRecord::Rollback,
+        "Primary attribute `#{params[:attribute_friendly_id]}` not found" if @primary_attribute.nil?
+
+      @extended_attributes = @primary_attribute.extended_attributes
+      @manually_sorted = @primary_attribute.manually_sorted?
```

Comment:
Do we need this ivar or do we just always check against `@primary_attribute.manually_sorted?`?

Reply:
That's what I had initially but changed it to this so I was just passing around a bool. Happy to change it back since it's not really providing any benefit

Reply:
Working on that as we speak :)